README.md

```diff
@@ -184,6 +185,87 @@ $ remme public-key get-list \
 }
 ```
 
+### Transaction
+
+Get list of transaction — ``remme transaction get-list``:
+
+| Arguments   | Type   |  Required | Description                                            |
+| :---------: | :----: | :-------: | -----------------------------------------------------  |
+| ids         | String |  No       | Identifiers to get a list of transactions by.          |
+| start       | String |  No       | Identifier to start paging (inclusive).                |
+| limit       | Integer|  No       | Maximum amount of transactions to return.              |
+| head        | String |  No       | Identifier of block's head.                            |
+| reverse     | Bool   |  No       | If transactions should be reversed.                    |
+| node-url    | String |  No       | Node URL to apply a command to.                        |
+| family-name | String |  No       | List transaction by its family name.                   |
+
+```bash
+$ remme transaction get-list \
+    --ids=64d032fbaae9bc59f9e5484ec6f52cbceef567923456039a26a1cfb8bc9ee2431ac2b5de43efce28ef11820a3734dab9fa56db57a1b2fbdc2323036cceeab6ab \
```

Comment:
Show the example of multiple ids, please.

Reply:
@dmytrostriletskyi, done.